pom.xml

```diff
@@ -786,6 +786,15 @@
 
     <build>
         <plugins>
+            <plugin>
```

Comment:
No need to add the compiler plugin as it is inherited from WSO2 carbon parent POM v5 (see https://github.com/wso2/carbon-parent/blob/carbon-parent-5/pom.xml#L216-L223)